examples/js/exporters/GLTFExporter.js

```diff
@@ -325,6 +325,26 @@ THREE.GLTFExporter.prototype = {
 
 		}
 
+		/**
+		 * Serializes a userData.
+		 *
+		 * @param {Object} userData
+		 * @returns {Object}
+		 */
+		function serializeUserData( userData ) {
+
+			try {
+
+				return JSON.parse( JSON.stringify( userData ) );
+
+			} catch {
```

Comment:
`} catch ( e ) {` is better otherwise we have errors like: https://travis-ci.org/mrdoob/three.js/builds/375969445#L1526

Reply:
Oops, I should've run `npm run test` before sending PR.

BTW, even without this PR I faced this error on `npm run test`. You, too?

>not ok 1 test/unit/three.source.unit.js > Failed to load the test file with error:
>SyntaxError: Unexpected token {
>    at exports.runInThisContext (vm.js:53:16)
>    at Module._compile (module.js:373:25)
>    at Object.Module._extensions..js (module.js:416:10)
>    at Module.load (module.js:343:32)
>    at Function.Module._load (module.js:300:12)
>    at Module.require (module.js:353:17)
>    at require (internal/module.js:12:17)
>    at run (C:\Users\takahiro\Documents\three.js\node_modules\qunit\bin\run.js:54:4)
>    at Object.<anonymous> >(C:\Users\takahiro\Documents\three.js\node_modules\qunit\bin\qunit:56:2)
>    at Module._compile (module.js:409:26)
>  message: "should be able to load file"
>  severity: failed
>  actual: false
>  expected: true
>  stack:     at Object.<anonymous> >(C:\Users\takahiro\Documents\three.js\node_modules\qunit\bin\run.js:61:13)
>    at runTest (C:\Users\takahiro\Documents\three.js\node_modules\qunit\qunit\qunit.js:1530:30)
>    at Object.run >(C:\Users\takahiro\Documents\three.js\node_modules\qunit\qunit\qunit.js:1516:6)
>    at C:\Users\takahiro\Documents\three.js\node_modules\qunit\qunit\qunit.js:1728:12
>    at advanceTaskQueue >(C:\Users\takahiro\Documents\three.js\node_modules\qunit\qunit\qunit.js:1129:6)
>    at advance (C:\Users\takahiro\Documents\three.js\node_modules\qunit\qunit\qunit.js:1110:4)
>  ...
>1..1
>pass 0
>skip 0
>todo 0
>fail 1

Reply:
Looks like it works fine on my machine. I have this output:
![image](https://user-images.githubusercontent.com/12612165/39714670-1b11fb66-522b-11e8-9942-64ee911e73ed.png)

Reply:
I see, it seems a problem in my environment. Thanks

Reply:
Updating node modules has solved, thanks.

Reply:
Great 👍